kuma/payments/jinja2/payments/thank_you.html

```diff
@@ -71,8 +74,61 @@ <h2 class="highlight highlight-spanned">
 
 {% block content %}
     <main id="contributions-page" class="contributions-page">
+        <!-- TODO: replace with waffle flag/varible -->
+        {% if False %}
+            <section class="section">
+                <div class="index">
+                    <h2>{{_('Useful Things')}}</h2>
+                </div>
+                <div class="faqs clear feedback">
+                    <div class="faq-feedback">
+                        <div class="sub-section">
+                            <h3>{{_('Managing your payments')}}</h3>
+                            <p>
+                                {% trans url="mailto:mdn-support@mozilla.com?Subject=Manage%20monthly%20payment" %}
```

Comment:
This should use the setting instead:
```suggestion
                                {% trans url="mailto:" + settings.CONTRIBUTION_SUPPORT_EMAIL + ?Subject=Manage%20monthly%20payment",
                                         email=settings.CONTRIBUTION_SUPPORT_EMAIL  %}
```

Maybe it is a good time to change the name of the setting to ``PAYMENTS_SUPPORT_EMAIL`` as well,